### DISCUSSION TITLE
Bot is not working for me on the Windows 11 Localhost

### DISCUSSION THREAD

**Discussion by delidolu1adam** [bongo-devs/lavamusic#503] [General]
I followed the installation instructions completely, but even though the bot is online on the server, I keep getting this error when I use a command.

-----

An error occurred: PrismaClientKnownRequestError: 
Invalid prisma.guild.findUnique() invocation:

Raw query failed. Code: unknown. Message: Kind: Command failed: Error code 8000 (AtlasError): empty database name not allowed
    at 1273cd53db701c6ff32c53c578b3f30c/atlasproxy/session_proxy.go:708
    at 1273cd53db701c6ff32c53c578b3f30c/atlasproxy/session_proxy.go:1614
    at 1273cd53db701c6ff32c53c578b3f30c/atlasproxy/session_proxy.go:1189
    at 10gen/mongonet@v0.0.0-20240114100216-4b92ace08452/proxy_session.go:365
    at 10gen/mongonet@v0.0.0-20240114100216-4b92ace08452/proxy_session.go:118
    at 10gen/mongonet@v0.0.0-20240114100216-4b92ace08452/session.go:217
    at 10gen/mongonet@v0.0.0-20240114100216-4b92ace08452/server.go:373
    at src/runtime/asm_arm64.s:1197, labels: {}``
    
    -----
The above error has been fixed somehow. I have no idea how. Now when I use a command, the bot joins the channel and leaves immediately. Then it sends the following message:

An error occurred: Error: Can't find any nodes to connect on

**Comment by infnibor** [comment]
Use public node from https://lavainfo.moebot.pro or update the code to latest version 4.1.0. and run AutoNode in .env
